IFC4x3/Properties/r/RatedVoltage_0KkDpffYz8L86gzG31TCSi/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="RatedVoltage_0KkDpffYz8L86gzG31TCSi" Name="RatedVoltage" UniqueId="14b8dce9-a62f-4854-81aa-f500c174c72c" PrimaryDataType="IfcElectricVoltageMeasure" />
```

Comment:
Done!

Reply:
Leaving this open because the reference to the existing does not seem to be correct. The id of the existing RatedVoltage that I found is '0_oie0qSSHuO00025QrE$V'